lab_3_keywords_textrank/main.py

```diff
@@ -66,7 +75,8 @@ def _remove_stop_words(self, tokens: tuple[str, ...]) -> tuple[str, ...]:
             tuple[str, ...]
                 tokens without stop-words
         """
-        pass
+        cleaned_tokens = [word for word in tokens if word not in self._stop_words]
+        return tuple(cleaned_tokens)
```

Comment:
create the tuple and return it immediately.

Reply:
done